src/utils/helpers.ts

```diff
@@ -1,12 +1,59 @@
+import isCI from 'is-ci';
+import prompts from 'prompts';
+import { logger } from '../logger';
+
+const FALSY_ENV_VALUES = new Set(['', '0', 'false', 'no']);
```

Comment:
Moving of `promptConfirmation` to a helper function triggered a chain of refactors around `hasInput` and `isDryRun`. Apologies for the side quest 🙂